src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -613,71 +613,53 @@ private static FormulaValue SortOptionSet(List<(DValue<RecordValue> row, Formula
             return new InMemoryTableValue(irContext, pairs.Select(pair => pair.row));
         }
 
-        private static async Task<DValue<RecordValue>> LazyFilterRowAsync(
-           EvalVisitor runner,
-           EvalVisitorContext context,
-           DValue<RecordValue> row,
-           LambdaFormulaValue filter)
+        private static async IAsyncEnumerable<DValue<RecordValue>> FilterRowAsync(EvalVisitor runner, EvalVisitorContext context, DValue<RecordValue> row, LambdaFormulaValue filter, [EnumeratorCancellation] CancellationToken cts = default)
```

Comment:
before, this returned a single item; now it returns an enumerable.  Intentional?